### DISCUSSION TITLE
État du Fullscreen sauvegardé

### DISCUSSION THREAD

**Discussion by Arubinu** [skarab42/marv#215] [Ideas]
Bonjour,

Je relance de temps à autre Marv et je remarque qu'il prend plus de place que précédemment à chaque reprise.
En fait j'utilise Marv en Fullscreen, et au démarrage il perd ce paramètre et fini par dépasser de quelques pixels de chaque côté (il prend plus de place).

Je ne sais pas si c'est un mauvais fonctionnement de quelque chose déjà présent ou pas, mais je le précise parce que ça m'a intrigué ^^

**Comment by skarab42** [comment]
Merci pour le fix :)
